### PR TITLE
Add TeleopAutoScoreCommandGroup

### PR DESCRIPTION
**Pull request by Tyson453**

### CODE REVIEW

File: robotbase/src/main/java/com/team2357/frc2023/controls/SwerveDriveControls.java [hunk 1]
```diff
@@ -15,20 +16,35 @@ public class SwerveDriveControls {
...
+    public void mapControls() {
+        // Zero swerve drive
         m_backButton.whileTrue(new InstantCommand(() -> SwerveDriveSubsystem.getInstance().zeroGyroscope()));
+        
+        // Intake commands
         m_rightBumper.whileTrue(new RunIntakeCommand());
         m_leftBumper.whileTrue(new ReverseIntakeCommand());
 
+        // Teleop auto
+        m_aButton.onTrue(new TeleopAutoScoreCommandGroup());
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/com/team2357/frc2023/controls/SwerveDriveControls.java:47]: Let's use a trigger so the driver does not have to let go of the sticks.

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]
This is a good initial thought process of what we need to do. Good work, We will have to rework it to actually incorporate the robot pose (need to discuss with @legokid on what the network table topic will be). We will also have to add the creation of the trajectory commands so we can auto score on the entire community.

File: robotbase/src/main/java/com/team2357/frc2023/controls/SwerveDriveControls.java (hunk 1 above)
Comment (legokid) [reply robotbase/src/main/java/com/team2357/frc2023/controls/SwerveDriveControls.java:47]: Here is what I would suggest for the json format to be passed via networktables
https://github.com/frc2357/FRC-2023/blob/add/wpilibpi-test/wpilibpi/apriltags_nt.json

Comment (NolanCampbell2357) [reply robotbase/src/main/java/com/team2357/frc2023/controls/SwerveDriveControls.java:47]: Thanks, we'll make this work.

**Review by NolanCampbell2357 — APPROVED** [review]

**Pull request merged**